isis/src/control/apps/jigsaw/jigsaw.cpp

```diff
@@ -0,0 +1,546 @@
+#include "jigsaw.h"
```

Comment:
nitpick but this should probably be moved as the last include, since most other conversion do the same.

Reply:
Thanks for pointing this out!